analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -151,6 +151,38 @@ def add_arguments_to_parser(parser):
                              "reports and overwrites only those files that "
                              "were update by the current build command).")
 
+    parser.add_argument('--compile-uniqueing',
+                        type=str,
+                        dest="compile_uniqueing",
+                        default="none",
+                        required=False,
+                        help="Specify the method the compilation"
+                             " actions in the  compilation database are"
```

Comment:
Once you start a line with a space, once you insert the space to the end of the line. Do not mix it. Please insert spaces to the end of the lines.